sdk/src/main/java/com/mapbox/maps/renderer/widget/Widget.kt

```diff
@@ -40,4 +49,9 @@ abstract class Widget internal constructor() {
    * Get absolute rotation of widget in degrees.
    */
   abstract fun getRotation(): Float
+
+  @JvmSynthetic
+  internal fun setRepaintTrigger(trigger: () -> Unit) {
```

Comment:
if we rename to `invalidate` this maybe could be called `addOnInvalidateListener`.

Reply:
we did not use `Trigger` anywhere else and I'm not sure I did see such naming before...
perhaps `doOnRepaint`?

Reply:
renamed to `doOnRepaint`